app/src/cpp/cfc-cpp/jni.cpp

```diff
@@ -50,7 +51,11 @@ namespace {
 		int guideOffset = minsz >> 5;
 		int outlineOffset = (outlineWidth - guideWidth) >> 1;
 
-		cv::Scalar color = in_progress? cv::Scalar(0,255,0) : cv::Scalar(255,255,255);
+		cv::Scalar color = cv::Scalar(255,255,255);
+		if (in_progress == 1)
+			color = cv::Scalar(255,100,100);
```

Comment:
Adding visual feedback for when extracts are happening, but decodes aren't.

The color is red right now -- I might change it. (green = success)